source/braille.py

```diff
@@ -2346,6 +2346,7 @@ def initialize():
 	log.info("Using liblouis version %s" % louis.version())
 	import serial
 	log.info("Using pySerial version %s"%serial.VERSION)
+	bdDetect.initializeDetectionData()

```

Comment:
when resolving this merge conflict, I kept this line where it currently is in master. However, it might need to get moved to `handlePostConfigProfileSwitch`?

Reply:
Why do you think it should be there? I don't think it needs to re-initialize after every config update?